src/Database/index.js

```diff
@@ -126,6 +134,23 @@ export default class Database {
     return merge$(...changesSignals).pipe(startWith(null))
   }
 
+  _subscribers: Array<[TableName<any>[], () => void]> = []
+
+  // Notifies `subscriber` on change in any of passed tables (only a signal, no change set)
+  experimentalSubscribe(tables: TableName<any>[], subscriber: () => void): () => void {
+    if (!tables.length) {
+      return noop
```

Comment:
if you don't pass an inline array to `experimentalSubscribe`, but just pass a variable from some other code, you could get there… not sure why, but i also don't see that it would be harmful